### ISSUE TITLE
Battery optimization issue

### ISSUE THREAD

**Issue by Dr-Fate** [vfsfitvnm/ViMusic#798]
### Steps to reproduce the bug

Battery optimization causes the app stop working in a long pause.

1) open the app and listen something
2) pause for a few mins (for ex: make a coffee, talk to someone, take a deep breath, think about the future...)
3) hit the play button

### Expected behavior

The song should resume and continue playing

### Actual behavior

The player in the notification bar goes black and the play button doesn't resume the song

### ViMusic version

0.5.4

### Android version

Android 13

### Additional information

The workaround is to set the battery permission to "unrestricted" for the app and works like a charm.
My device is Pixel 6

**Comment by vfsfitvnm** [comment]
Yeah, this is how it's supposed to work for now (sadly). You can learn more in _Settings > Other > Service Lifetime_

**Comment by Dr-Fate** [comment]
Is not a big deal, the app is great.
Thanks for the info
